enterprise/j2ee.dd/licenseinfo.xml

```diff
@@ -77,6 +77,7 @@
         <file>src/org/netbeans/modules/j2ee/dd/impl/resources/jsp_2_1.xsd</file>
         <file>src/org/netbeans/modules/j2ee/dd/impl/resources/jsp_2_2.xsd</file>
         <file>src/org/netbeans/modules/j2ee/dd/impl/resources/jsp_2_3.xsd</file>
+        <file>src/org/netbeans/modules/j2ee/dd/impl/resources/jsp_3_0.xsd</file>
```

Comment:
This is EPL-2.0 (or GPL2-CP, but lets EPL-2.0 should be preferred)

Reply:
Done.